docs/docs-app/components/app.js

```diff
@@ -1,14 +1,33 @@
-import React from 'react';
+import React, {Component} from 'react';
 
 import Header from './header';
 
 /* eslint-disable react/display-name*/
-export default ({children}) => (
-  <div className="app">
-    <Header />
-    <div className="app-container">
-      {children}
-    </div>
-  </div>
-);
+class App extends Component {
```

Comment:
generally in react-vis we try to avoid constructor calls, it would be preferable to simply say

```javascript
class App extends Component {
state = {
opacity: 1,
isMenuOpen: false
}
}
```

and then call toggle menu as an anon function, which is pretty reasonable given that's its only used once